docs/application/native/guides/ui/efl/wearable/component-ctxpopup.md

```diff
@@ -32,11 +32,14 @@ When shown, the ctxpopup automatically selects an area inside its parent object'
 
 The ctxpopup has the following styles for the rectangular screen:
 
-- `default`  
+- `default`
+
   The ctxpopup has the following styles for the circular screen:
 
 - `select_mode`
-- `select_mode/top`, `select_mode/bottom`These 2 styles can be used as a pair.
+
+- `select_mode/top`, `select_mode/bottom`
```

Comment:
line 41,42: The separated lines are merged as one line.(These 2 styles)
line 85: The separated lines are merged as one line. Code block.(elm_object_item_part_text_set(it, "default", "New label");)

Reply:
I think there are some wrong code tags, but I didn't fix them because the author thought the descriptions were correct.